test/verify/check-users

```diff
@@ -51,6 +51,11 @@ def createUser(browser, user_name, real_name, password, locked, force_password_c
         browser.set_checked('#accounts-create-force-password-change', force_password_change)
         browser.wait_visible('#accounts-create-locked:not(:checked)')
 
+    if expected_uid is not None:
+        browser.wait_visible(f'#accounts-create-user-uid[value="{expected_uid}"]')
```

Comment:
unnecessary f-string, expected_uid is already a string

Reply:
What do you mean? But we still have to check that the correct value is present? and "wait_in_text" would not work here I believe.

Reply:
Nevermind, I misread that, sorry!